### PR TITLE
Consistent formatting for sizeof(foo)

### PR DESCRIPTION
**Pull request by richsalz**
Spurred by the recent bug that had the "-1" inside the sizeof argument, this PR fixes all uses of sizeof to be conformant with our style guide.  Not going to cherry-pick back; I could do separate PR's for the other branches if someone feels strongly about wanting it.

### CODE REVIEW

File: test/ssltest_old.c [hunk 1]
```diff
@@ -1915,8 +1915,8 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
             if (cw_num > 0) {
                 /* Write to server. */
 
-                if (cw_num > (long)sizeof cbuf)
-                    i = sizeof cbuf;
+                if (cw_num > (long)sizeof(cbuf))
+                    i = (int)sizeof(cbuf);
```
Comment (levitte) [inline test/ssltest_old.c]: That cast should not be part of this PR.  They are also a subject of discussion, and should not be added spuriously.

File: test/ssltest_old.c [hunk 2]
```diff
@@ -1994,8 +1994,8 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
             if (sw_num > 0) {
                 /* Write to client. */
 
-                if (sw_num > (long)sizeof sbuf)
-                    i = sizeof sbuf;
+                if (sw_num > (long)sizeof(sbuf))
+                    i = (int)sizeof(sbuf);
```
Comment (levitte) [inline test/ssltest_old.c]: Remove the cast

File: test/ssltest_old.c [hunk 3]
```diff
@@ -2177,8 +2177,8 @@ int doit_biopair(SSL *s_ssl, SSL *c_ssl, long count,
             if (cw_num > 0) {
                 /* Write to server. */
 
-                if (cw_num > (long)sizeof cbuf)
-                    i = sizeof cbuf;
+                if (cw_num > (long)sizeof(cbuf))
+                    i = (int)sizeof(cbuf);
```
Comment (levitte) [inline test/ssltest_old.c]: Remove the cast

File: test/ssltest_old.c [hunk 4]
```diff
@@ -2256,8 +2256,8 @@ int doit_biopair(SSL *s_ssl, SSL *c_ssl, long count,
             if (sw_num > 0) {
                 /* Write to client. */
 
-                if (sw_num > (long)sizeof sbuf)
-                    i = sizeof sbuf;
+                if (sw_num > (long)sizeof(sbuf))
+                    i = (int)sizeof(sbuf);
```
Comment (levitte) [inline test/ssltest_old.c]: Remove the cast

**Comment by richsalz** [comment]
Actually, in all four of those cases the cast is safe and correct because it is the size of an on-stack 8K buffer.  But I removed them because it's not relevant to the main point of this PR.  Thanks for the nudge.  updating commit pushed.

**Comment by dot-asm** [comment]
This is apparently committed already, so it should be closed... Right?
